.github/workflows/deploy_prod.yaml

```diff
@@ -2,11 +2,20 @@ name: Deploy
 
 on:
   workflow_dispatch:
+    inputs:
+      branch:
+        description: "Branch to deploy"
+        required: true
+        default: "main"
```

Comment:
The branch is a default option as part of the workflow dispatch, and is unused below:

![image](https://github.com/populationgenomics/variant-curation-portal/assets/22381693/d59a1444-c166-489f-b532-d2a9bbf93686)